about.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Add a link to navigate back to home page from about page

Reply:
add change